DeskReserve/Data/DBContext/Entity/Building.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System.ComponentModel;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace DeskReserve.Data.DBContext.Entity
+{
+    [Table("building")]
+    public class Building
+    {
+        [Key]
```

Comment:
Why do you believe these are the maximum possible input

Reply:
I check the longest name of town and for the other variables i believe we don't need more size.